lib/testing/src/lib/core/utils/browser-actions.ts

```diff
@@ -191,4 +191,15 @@ export class BrowserActions {
         stream.end();
     }
 
+    static async pressCmd(): Promise<void> {
```

Comment:
It doesn't add any value wap a method in a method better just use it where is needed it reduces the code

Reply:
Removed one line methods.